build.gradle

```diff
@@ -327,7 +336,7 @@ project(':embedded-database-spring-test') {
 
                         eachDependency { details ->
                             if (details.requested.group == 'junit') {
-                                details.useVersion "4.12"
+                                details.useVersion "4.13.2"
```

Comment:
I just want to warn you in advance that I don't recommend upgrading to JUnit 5 here, because migrating the logic in TestAssumptions isn't trivial. I've tried it at least once before and wasn't successful.